aldryn_redirects/managers.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import models
+
+from six.moves.urllib.parse import urlparse, parse_qsl
+
+
+class StaticRedirectManager(models.QuerySet):
+    def get_for_request(self, request):
+        path_info = request.path_info
+        # request.GET sounds tempting below but wouldn't work for malformed querystrings (such as '/path?hamster').
+        request_query_params = dict(parse_qsl(urlparse(request.get_full_path()).query, keep_blank_values=True))
+
+        candidates = self.filter(sites__id__exact=settings.SITE_ID, inbound_route=path_info)
```

Comment:
shouldn't this be in an `else` below?

Reply:
Result is the same, right?

I prefer this approach, it's more concise. Performance is the same since querysets have lazy evaluation.

Reply:
Yes but is confusing.. like why create a queryset if it won't be used.
Anyways is ok for me.